cmd/bundle/summary.go

```diff
@@ -132,18 +132,6 @@ func prepareBundleForSummary(cmd *cobra.Command, forcePull, includeLocations boo
 
 	if forcePull || noCache {
 		bundle.ApplyContext(ctx, b, statemgmt.StatePull())
-
-		if logdiag.HasError(ctx) {
-			return nil
-		}
-
-		if !b.DirectDeployment {
-			bundle.ApplySeqContext(ctx, b,
-				terraform.Interpolate(),
-				terraform.Write(),
```

Comment:
I think since we removed this line `bundle summary` might fail on the fresh bundles (which were never deployed before). Do we have a test for this?